template.md

```diff
@@ -3,12 +3,12 @@
 
 ### Previous Meeting minutes:
 
-- http://github.com/gluster/community/meetings
+- https://github.com/gluster/community/tree/master/meetings
```

Comment:
Instead of a raw link, maybe we can use a tag for this.. something similar to that of community calendar link provided below.

Reply:
Yeah that's better.